app/Http/Controllers/Backend/WikiController.php

```diff
@@ -7,28 +7,23 @@
 use Illuminate\Http\Request;
 
 class WikiController extends Controller {
-    private static $with = ['wikiDb', 'wikiQueryserviceNamespace', 'settings'];
-
     public function getWikiForDomain(Request $request): \Illuminate\Http\JsonResponse {
-        $domain = $request->input('domain');
+        $validated = $request->validate([
+            'domain' => 'required|string',
+        ]);
 
+        $domain = $validated['domain'];
         // XXX: this same logic is in quickstatements.php and platform api WikiController backend
         try {
-            if ($domain === 'localhost' || $domain === 'mediawiki') {
```

Comment:
For tom: is this really safe to remove? Go and quickly check where it might be used.

Reply:
I think this was for special curl handling in case of running in the docker compose setup https://github.com/wbstack/magnustools/pull/18

I don't think usage of magnustools is playing a role in creating the mw/wikibase sql schema